_posts/2015-03-16-CocoaPods.org-Take-Two.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "CocoaPods.org Take Two"
+author: orta
+categories: cocoapods.org releases website
+---
+
+TL;DR: [CocoaPods.org][1] version 2.1 has been released, with support for inline deep search results, and pod pages.
+
+<!-- more -->
+
+<style>
+  /* for the CP preview */
+.container > .row > .content > center > a > img {
+  border: 1px solid black;
+}
+</style>
+
+## Backstory.
+
+The CocoaPods web site is an interesting beast. In October 2013, we set around [rewriting][2] the CocoaPods web pages. We centralized the documentation, created a blog and redesigned everything. 
+
+We settled on using a mix of middleman pages and jekyll. Given that, at the time, there was no central database for CocoaPods, it made sense to try and keep it as a static site for speed and maintainability.
+
+Once [CocoaDocs v2][3] launched in May 2014, I set my sights on trying to move cocoapods.org a little bit closer to my larger vision of what the website would provide.
+
+In an ideal world, you should be able to find a library that fits your constraints using just the front page of CocoaPods.org. Often there is more than one library, and that’s where things gets a bit more complex. In previous versions of the site, in order to choose between multiple libraries, you need to open them up on GitHub in separate tabs and compare. Now, you don’t need to. 🎉
+
+## Here’s how it works
+
+Let’s take a random pod. [MiniFuture][4]. This is what happens after a Pod is released:
+
+* A new Podspec version is added to [Trunk][5]
+* A webhook from trunk triggers a reindex of the pod in the search engine which uses the generated metadata to sort and display pod results.
+* A another webhook gets sent to CocoaDocs triggering a documentation build
+* The documentation build uses [Jazzy][jazzy] ( or [appledoc][appledoc] for Objective-C projects ) to generate documentation. 
+* After documentation has finished CocoaDocs starts to generating metadata. This provides us with a rich dataset to start providing some useful statistics. This is centralized in [metrics.cocoapods.org][6], where we currently hold up-to-date GitHub metrics also.
```

Comment:
"After documentation has finished" kind of unnecessary since this is assumed to be a list of what happens, in order